### ISSUE TITLE
EDC Artifact Consumption in MVD repositories

### ISSUE THREAD

**Issue by Izzzu** [agera-edc/MinimumViableDataspace#167]
Feature #30 
After #164 

## Description
We want to make sure that all repositories have dependencies to publicly available EDC libraries instead of using locally published artefacts.

## Acceptance criteria
- [ ] The Registration Service uses published version of EDC instead of checking out the repository and publishing locally the artefacts
- [ ] The Identity Hub uses published version of EDC instead of checking out the repository and publishing locally the artefacts
- [ ] The MinimumViableDataspace repo uses published version of EDC instead of checking out the repository and publishing locally the artefacts

**Comment by algattik** [comment]
@zeier the DataDashboard repo does not use EDC, AFAIK

**Comment by zeier** [comment]
@algattik correct, it even must not use EDC. @Izzzu, I removed the 4th acceptance criteria.

**Comment by zeier** [comment]
Put on hold, as it is projected to be too much scope for the team until end of August.
#36 and #157 have higher priority.
This item can be taken up again if there is capacity.
